app/screens/AppNavigator/screens/Settings/SettingsScreen.tsx

```diff
@@ -207,6 +222,28 @@ function RowExitWalletItem (): JSX.Element {
   )
 }
 
+function PrivacyLockToggle ({
+  value,
+  onToggle
+}: { disabled?: boolean, value: boolean, onToggle: (newValue: boolean) => void }): JSX.Element {
+  return (
+    <ThemedView
+      light={tailwind('bg-white border-b border-gray-200')}
+      dark={tailwind('bg-gray-800 border-b border-gray-700')}
+      style={tailwind('flex p-4 pr-2 flex-row items-center justify-between')}
+    >
+      <ThemedText testID='text_privacy_lock' style={tailwind('font-medium')}>
+        {translate('screens/Settings', 'Privacy Lock')}
```

Comment:
@fuxingloh we have discussed it before, but we decided not to put any OS specific features (e.g, FaceID/TouchID for Apple etc.) since it might affect store approval.

Reply:
https://developer.apple.com/design/human-interface-guidelines/ios/user-interaction/authentication/

> Always identify the authentication method. A button for signing in to your app using Face ID, for example, should be titled "Sign In with Face ID" rather than "Sign In."

Their guidelines require us to mention it for Apple.